modin/pandas/test/utils.py

```diff
@@ -663,6 +665,27 @@ def assert_dtypes_equal(df1, df2):
                 break
 
 
+def sort_data(data):
+    """Sort the passed sequence."""
+    if isinstance(data, (pandas.DataFrame, pd.DataFrame)):
+        return data.sort_values(data.columns.to_list(), ignore_index=True)
+    elif isinstance(data, (pandas.Series, pd.Series)):
+        return data.sort_values()
+    else:
+        return np.sort(data)
+
+
+def sort_if_range_partitioning(df1, df2, comparator=None):
```

Comment:
Do we have a note somewhere that range partitioning gives rows mismatch?

Reply:
Not yet, it should be done in #6987